GCEWindowsAgent/accounts.go

```diff
@@ -158,20 +160,35 @@ func createcredsJSON(k windowsKeyJSON, pwd string) (*credsJSON, error) {
 
 type accounts struct {
 	newMetadata, oldMetadata *metadataJSON
+	config                   *ini.File
 }
 
 func (a *accounts) diff() bool {
 	return !reflect.DeepEqual(a.newMetadata.Instance.Attributes.WindowsKeys, a.oldMetadata.Instance.Attributes.WindowsKeys)
 }
 
-func (a *accounts) disabled() bool {
-	d := a.newMetadata.Instance.Attributes.DisableAccountManager
-	if d != accountDisabled {
-		accountDisabled = d
-		logStatus("account", d)
-	}
+func (a *accounts) disabled() (disabled bool) {
+	defer func() {
+		if disabled != accountDisabled {
+			accountDisabled = disabled
+			logStatus("account", disabled)
+		}
+	}()
 
-	return d
+	var err error
+	disabled, err = strconv.ParseBool(a.config.Section("accountManager").Key("disable").String())
+	if err == nil {
+		return disabled
+	}
+	disabled, err = strconv.ParseBool(a.newMetadata.Instance.Attributes.DisableAccountManager)
```

Comment:
I thought about it and decided on no, I don't want to spam the logs just because of a bad configs. We could add a check to only log errors once but I thought that would be added complexity (along with the checking for empty string) that may not gain us anything. If we find errors like that useful we can always add it in later.